### PR TITLE
new icon: apachekafka (original, original-wordmark)

### PR DESCRIPTION
**Pull request by tupaschoal**
`original`:
apachekafka_original
`original-wordmark`:
apachekafka_original-wordmark

### CODE REVIEW

File: icons/apachekafka/apachekafka-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...22 3.22 0 002.535-.332l.043-.02a3.403 3.403 0 001.52-2.043 3.471 3.471 0 00-.329-2.59M48.588 53.25h3.726v11.578l5.375-6.05h4.47l-6.49 7.007 6.517 8.883h-4.27l-5.602-7.934v7.934h-3.726V53.25M66.486 66.793c0 1.969 1.39 4.836 4.75 4.836 2.075 0 3.442-1.102 4.153-2.547.37-.7.539-1.45.597-2.227a5.015 5.015 0 00-.43-2.234c-.652-1.5-2.1-2.805-4.347-2.805-3.016 0-4.723 2.489-4.723 4.95v.027zm13.196 7.875h-3.727v-2.285c-.992 1.847-3.07 2.773-5.262 2.773-5.004 0-7.933-3.965-7.933-8.449 0-5.004 3.554-8.418 7.933-8.418 2.844 0 4.578 1.531 5.262 2.809v-2.32h3.727v15.89M83.9 61.703h-1.93v-2.926h1.93v-.894c0-4.723 3.532-5.004 6.204-5.035v2.918c-.938 0-2.473 0-2.473 2.234v.777h2.473v2.926H87.63v12.965H83.9V61.703M92.639 53.25h3.726v11.578l5.375-6.05h4.465l-6.484 7.007 6.511 8.883h-4.265l-5.602-7.934v7.934H92.64V53.25M110.533 66.793c0 1.969 1.395 4.836 4.75 4.836 2.074 0 3.442-1.102 4.153-2.547.37-.7.543-1.45.597-2.227a5.015 5.015 0 00-.43-2.234c-.652-1.5-2.101-2.805-4.347-2.805-3.012 0-4.723 2.489-4.723 4.95v.027zm13.2 7.875h-3.727v-2.285c-.996 1.847-3.07 2.773-5.262 2.773-5.008 0-7.933-3.965-7.933-8.449 0-5.004 3.554-8.418 7.933-8.418 2.844 0 4.578 1.531 5.262 2.809v-2.32h3.726v15.89"/></g></svg>
```
Comment (Panquesito7) [inline icons/apachekafka/apachekafka-original-wordmark.svg:1]:
```suggestion
<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g fill="#231F20"><path d="M15.885 41.203a3.29 3.29 0 00-2.371-.996c-.93 0-1.762.379-2.356.996h-.015a3.418 3.418 0 00-.973 2.406c0 .95.371 1.797.973 2.403l.015.015a3.256 3.256 0 002.356.989 3.3 3.3 0 002.37-.989l.009-.015a3.388 3.388 0 00.972-2.403c0-.941-.37-1.797-.972-2.406h-.008zm-2.371 46.59c.933 0 1.77-.387 2.37-.992l.009-.016a3.39 3.39 0 00.972-2.394c0-.95-.37-1.801-.972-2.414h-.008a3.275 3.275 0 00-2.371-.997c-.93 0-1.762.375-2.356.997h-.015a3.43 3.43 0 00-.973 2.414c0 .937.371 1.789.973 2.394l.015.016a3.29 3.29 0 002.356.992zM31.736 77.48a3.296 3.296 0 002.035-1.582l.063-.117c.406-.777.48-1.672.266-2.465a3.363 3.363 0 00-1.563-2.074l-.047-.031a3.281 3.281 0 00-2.488-.305 3.278 3.278 0 00-2.035 1.586 3.442 3.442 0 001.222 4.656h.004c.801.47 1.703.555 2.543.332zM16.943 60.512a4.808 4.808 0 00-3.43-1.45 4.8 4.8 0 00-3.425 1.45 4.958 4.958 0 00-1.41 3.484c0 1.363.539 2.594 1.41 3.496a4.805 4.805 0 006.855 0 5 5 0 001.414-3.496c0-1.367-.539-2.594-1.414-3.484zm-1.93-5.801a9.162 9.162 0 015.032 2.644h.012c.21.215.418.454.609.692l3.34-1.965a7.366 7.366 0 01-.11-4.18 7.278 7.278 0 013.364-4.449l.054-.039a7.078 7.078 0 015.418-.707 7.235 7.235 0 014.383 3.426v.004a7.464 7.464 0 01.723 5.566 7.283 7.283 0 01-3.363 4.457l-.457.274h-.047a7.13 7.13 0 01-4.973.46 7.179 7.179 0 01-3.496-2.18l-3.332 1.962a9.615 9.615 0 010 6.652l3.332 1.957a7.102 7.102 0 013.496-2.183 7.006 7.006 0 015.477.738l.109.055v.003a7.314 7.314 0 013.254 4.403 7.454 7.454 0 01-.723 5.558l-.062.125-.004-.011a7.295 7.295 0 01-4.313 3.32 7.156 7.156 0 01-5.476-.734v-.016a7.307 7.307 0 01-3.364-4.453 7.34 7.34 0 01.11-4.172l-3.34-1.961a9.204 9.204 0 01-.61.68l-.011.011a9.103 9.103 0 01-5.031 2.63v3.925c1.386.297 2.629 1 3.601 1.992l.008.008a7.375 7.375 0 012.11 5.188 7.358 7.358 0 01-2.11 5.171l-.008.024c-1.312 1.324-3.113 2.148-5.101 2.148a7.16 7.16 0 01-5.094-2.148h-.008v-.024A7.361 7.361 0 016.3 84.391c0-2.024.808-3.864 2.113-5.188v-.008h.008a7.137 7.137 0 013.598-1.992v-3.926a9.092 9.092 0 01-5.028-2.629l-.011-.011a9.46 9.46 0 01-2.711-6.64 9.467 9.467 0 012.71-6.642h.012a9.164 9.164 0 015.028-2.644V50.8a7.261 7.261 0 01-3.598-1.996h-.008v-.012A7.383 7.383 0 016.3 43.609c0-2.011.808-3.855 2.113-5.183l.008-.004a7.136 7.136 0 015.094-2.156c1.988 0 3.789.824 5.101 2.156v.004h.008a7.393 7.393 0 012.11 5.183 7.38 7.38 0 01-2.11 5.184l-.008.012a7.256 7.256 0 01-3.601 1.996v3.91zm18.758-2.61l-.027-.039a3.366 3.366 0 00-2.008-1.546 3.301 3.301 0 00-2.547.34h.004a3.36 3.36 0 00-1.562 2.07 3.45 3.45 0 00.336 2.586l.015.02c.461.8 1.2 1.34 2.02 1.554a3.22 3.22 0 002.535-.332l.043-.02a3.403 3.403 0 001.52-2.043 3.471 3.471 0 00-.329-2.59M48.588 53.25h3.726v11.578l5.375-6.05h4.47l-6.49 7.007 6.517 8.883h-4.27l-5.602-7.934v7.934h-3.726V53.25M66.486 66.793c0 1.969 1.39 4.836 4.75 4.836 2.075 0 3.442-1.102 4.153-2.547.37-.7.539-1.45.597-2.227a5.015 5.015 0 00-.43-2.234c-.652-1.5-2.1-2.805-4.347-2.805-3.016 0-4.723 2.489-4.723 4.95v.027zm13.196 7.875h-3.727v-2.285c-.992 1.847-3.07 2.773-5.262 2.773-5.004 0-7.933-3.965-7.933-8.449 0-5.004 3.554-8.418 7.933-8.418 2.844 0 4.578 1.531 5.262 2.809v-2.32h3.727v15.89M83.9 61.703h-1.93v-2.926h1.93v-.894c0-4.723 3.532-5.004 6.204-5.035v2.918c-.938 0-2.473 0-2.473 2.234v.777h2.473v2.926H87.63v12.965H83.9V61.703M92.639 53.25h3.726v11.578l5.375-6.05h4.465l-6.484 7.007 6.511 8.883h-4.265l-5.602-7.934v7.934H92.64V53.25M110.533 66.793c0 1.969 1.395 4.836 4.75 4.836 2.074 0 3.442-1.102 4.153-2.547.37-.7.543-1.45.597-2.227a5.015 5.015 0 00-.43-2.234c-.652-1.5-2.101-2.805-4.347-2.805-3.012 0-4.723 2.489-4.723 4.95v.027zm13.2 7.875h-3.727v-2.285c-.996 1.847-3.07 2.773-5.262 2.773-5.008 0-7.933-3.965-7.933-8.449 0-5.004 3.554-8.418 7.933-8.418 2.844 0 4.578 1.531 5.262 2.809v-2.32h3.726v15.89"/></g></svg>

```

File: icons/apachekafka/apachekafka-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...7 16.672 0 7.91 5.555 14.539 12.969 16.238v9.547c-10.117 1.773-17.84 10.59-17.84 21.191 0 10.652 7.797 19.5 17.992 21.211V95c-7.492 1.64-13.12 8.309-13.12 16.273 0 9.196 7.495 16.672 16.706 16.672 9.207 0 16.703-7.476 16.703-16.672 0-7.964-5.629-14.632-13.117-16.273V84.914a21.592 21.592 0 0013.133-7.625l7.902 5.586a16.45 16.45 0 00-.687 4.688c0 9.195 7.496 16.671 16.707 16.671 9.207 0 16.703-7.476 16.703-16.671 0-9.196-7.496-16.672-16.703-16.672zm0-38.984c4.465 0 8.097 3.63 8.097 8.086 0 4.453-3.632 8.082-8.097 8.082-4.469 0-8.102-3.629-8.102-8.082 0-4.457 3.633-8.086 8.102-8.086zm-48.742-15.18c0-4.456 3.632-8.081 8.101-8.081 4.465 0 8.098 3.625 8.098 8.082 0 4.457-3.633 8.082-8.098 8.082-4.469 0-8.101-3.625-8.101-8.082zm16.199 94.547c0 4.457-3.633 8.082-8.098 8.082-4.469 0-8.101-3.625-8.101-8.082 0-4.457 3.632-8.082 8.101-8.082 4.465 0 8.098 3.625 8.098 8.082zm-8.102-36.296c-6.226 0-11.293-5.059-11.293-11.274 0-6.219 5.067-11.277 11.293-11.277 6.23 0 11.297 5.058 11.297 11.277 0 6.215-5.066 11.274-11.297 11.274zm40.645 20.668c-4.469 0-8.102-3.625-8.102-8.082 0-4.458 3.633-8.083 8.102-8.083 4.465 0 8.097 3.625 8.097 8.082 0 4.458-3.632 8.083-8.097 8.083zm0 0" fill="#231f20"/></svg>
```
Comment (Panquesito7) [inline icons/apachekafka/apachekafka-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M86.758 70.89c-4.992 0-9.465 2.208-12.528 5.68l-7.851-5.547a21.275 21.275 0 001.312-7.32c0-2.531-.46-4.95-1.27-7.203l7.837-5.488c3.062 3.457 7.523 5.652 12.5 5.652 9.207 0 16.703-7.48 16.703-16.672 0-9.195-7.496-16.672-16.703-16.672-9.211 0-16.707 7.477-16.707 16.672 0 1.645.25 3.23.699 4.735l-7.84 5.488a21.578 21.578 0 00-13.36-7.746v-9.43c7.567-1.586 13.27-8.293 13.27-16.312C62.82 7.53 55.324.055 46.117.055c-9.21 0-16.707 7.476-16.707 16.672 0 7.91 5.555 14.539 12.969 16.238v9.547c-10.117 1.773-17.84 10.59-17.84 21.191 0 10.652 7.797 19.5 17.992 21.211V95c-7.492 1.64-13.12 8.309-13.12 16.273 0 9.196 7.495 16.672 16.706 16.672 9.207 0 16.703-7.476 16.703-16.672 0-7.964-5.629-14.632-13.117-16.273V84.914a21.592 21.592 0 0013.133-7.625l7.902 5.586a16.45 16.45 0 00-.687 4.688c0 9.195 7.496 16.671 16.707 16.671 9.207 0 16.703-7.476 16.703-16.671 0-9.196-7.496-16.672-16.703-16.672zm0-38.984c4.465 0 8.097 3.63 8.097 8.086 0 4.453-3.632 8.082-8.097 8.082-4.469 0-8.102-3.629-8.102-8.082 0-4.457 3.633-8.086 8.102-8.086zm-48.742-15.18c0-4.456 3.632-8.081 8.101-8.081 4.465 0 8.098 3.625 8.098 8.082 0 4.457-3.633 8.082-8.098 8.082-4.469 0-8.101-3.625-8.101-8.082zm16.199 94.547c0 4.457-3.633 8.082-8.098 8.082-4.469 0-8.101-3.625-8.101-8.082 0-4.457 3.632-8.082 8.101-8.082 4.465 0 8.098 3.625 8.098 8.082zm-8.102-36.296c-6.226 0-11.293-5.059-11.293-11.274 0-6.219 5.067-11.277 11.293-11.277 6.23 0 11.297 5.058 11.297 11.277 0 6.215-5.066 11.274-11.297 11.274zm40.645 20.668c-4.469 0-8.102-3.625-8.102-8.082 0-4.458 3.633-8.083 8.102-8.083 4.465 0 8.097 3.625 8.097 8.082 0 4.458-3.632 8.083-8.097 8.083zm0 0" fill="#231f20"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -129,6 +129,33 @@
         "color": "#303284",
         "aliases": []
     },
+    {
+        "name": "apachekafka",
+        "tags": [
+            "streaming"
```
Comment (Panquesito7) [inline devicon.json:135]:
```suggestion
            "streaming",
            "open-source"
```

**Review by Panquesito7 — COMMENTED** [review]
👍 Good work.
I see this icon as brown-black, but I see the icon completely black on their website. Do you think you could change it to black? Thanks.

**Comment by tupaschoal** [comment]
> I see this icon as brown-black, but I see the icon completely black on their website. Do you think you could change it to black?

It looks black, but it's brownish, take a look at the logo I got directly from https://kafka.apache.org/ by doing web code inspection: https://kafka.apache.org/logos/kafka_logo--simple.png

**Review by Panquesito7 — APPROVED** [review]
Great work, @tupaschoal! Thanks for your contribution! 👍 🎉

**Pull request merged**